StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -26,11 +26,11 @@ extension PaymentSheet {
         paymentHandler: STPPaymentHandler,
         isFlowController: Bool = false,
         paymentMethodID: String? = nil,
-        completion: @escaping (PaymentSheetResult) -> Void
+        completion: @escaping (PaymentSheetResult, STPAnalyticsClient.DeferredIntentConfirmationType?) -> Void
     ) {
         // Translates a STPPaymentHandler result to a PaymentResult
-        let paymentHandlerCompletion: (STPPaymentHandlerActionStatus, NSObject?, NSError?) -> Void = { status, _, error in
-            completion(makePaymentSheetResult(for: status, error: error))
+        let paymentHandlerCompletion: (STPPaymentHandlerActionStatus, STPAnalyticsClient.DeferredIntentConfirmationType?, NSObject?, NSError?) -> Void = { status, deferredIntentConfirmationType, _, error in
```

Comment:
I think we can delete `NSObject?` from the arguments

Reply:
D we ever pass a non-nil value for the confirmation type?  Seems like we can remove this argument (in which case you can probably revert all the paymentHandlerCompletion changes in this PR, and just pass nil in the line below this one.